tests/mypy/test_static_type_checking.py

```diff
@@ -139,6 +139,23 @@ def test_pandera_runtime_errors(fn) -> None:
     }
 ]
 
+PANDAS_SERIES_ERRORS_PLUGIN = [
```

Comment:
@cosmicBboy This change is caused by the release of a new `pandas-stubs` couple days back. I'm not familiar with the Mypy plugin, so I changed the error messages so it will pass; not sure if there's any fix required.